.scripts/build_steps.sh

```diff
@@ -6,7 +6,7 @@
 # benefit from the improvement.
 
 set -xeuo pipefail
-
+git config --global safe.directory '*'
```

Comment:
Am hoping we don't need this, but we can leave it for now. Though let's revisit when things are further along

Reply:
We don't.  It was needed for my local docker testing and snuck in a commit.